src/components/GatingStatus.tsx

```diff
@@ -77,7 +77,9 @@ export const ArtifactGreenwaveStatesSummary: React.FC<
     const satisfied = decision?.satisfied_requirements?.length;
     const unSatisfied = decision?.unsatisfied_requirements?.length;
     if (satisfied) {
-        reqSummary['ok'] = satisfied;
+        // Decrease the satisfied counter by 1 since the 'fetched-gating-yaml' task isn't displayed in UI.
+        // A similar thing is done in ArtifactStatesList() - `if (stateName === 'fetched-gating-yaml') continue;`
```

Comment:
Can you please use the ANSI C-style comment for comment spanning multiple lines?
```suggestion
        /*
         * Decrease the satisfied counter by 1 since the 'fetched-gating-yaml' task isn't displayed in UI.
         * A similar thing is done in ArtifactStatesList() - `if (stateName === 'fetched-gating-yaml') continue;`
         */
```